rest/client.go

```diff
@@ -16,6 +18,13 @@ type GetComponentsFilter struct {
 
 type Client interface {
 	GetComponentsByAsset(context.Context, uuid.UUID, GetComponentsFilter) ([]models.Component, error)
+
+	GetComponentRelations(context.Context, uuid.UUID) ([]models.Relation, error)
+	GetComponentRelationsPage(context.Context, uuid.UUID, int, string) (models.GetComponentRelationsResponse, error)
```

Comment:
I'm happy to merge this if we just make the ...Page functions non-public. Most consumers should not need them.

Reply:
Makes sense! :) Updated.